{"name": "test_broken", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1' == 1", "trace": "self = <python.test_allured.TestLingo object at 0x7fe636e28e80>\n\n    @allure.story(\"密码缺失\")\n    def test_broken(self):\n        #标注测试步骤\n        with allure.step(\"点击用户名\"):\n            print(\"输入用户名\")\n        with allure.step(\"点击密码\"):\n            print(\"输入密码\")\n        print(\"点击登录\")\n        with allure.step(\"点击登录会后登录失败\"):\n>           assert \"1\" == 1\nE           AssertionError: assert '1' == 1\n\ntest_allured.py:39: AssertionError"}, "steps": [{"name": "点击用户名", "status": "passed", "start": 1616237825699, "stop": 1616237825699}, {"name": "点击密码", "status": "passed", "start": 1616237825699, "stop": 1616237825699}, {"name": "点击登录会后登录失败", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1' == 1\n", "trace": "  File \"/Users/giod/PycharmProjects/testpro0/python/test_allured.py\", line 39, in test_broken\n    assert \"1\" == 1\n"}, "start": 1616237825699, "stop": 1616237825699}], "attachments": [{"name": "stdout", "source": "c6982094-c7d4-40aa-9dc6-f8b3a3a0a1e3-attachment.txt", "type": "text/plain"}], "start": 1616237825698, "stop": 1616237825699, "uuid": "b03a4e40-88f9-4cce-be5b-63d80bf57917", "historyId": "a567462599235da441aa31ae07d8e1ac", "testCaseId": "b229b5343521255f24c64ed5e4d6548e", "fullName": "test_allured.TestLingo#test_broken", "labels": [{"name": "story", "value": "密码缺失"}, {"name": "feature", "value": "登录成功"}, {"name": "suite", "value": "test_allured"}, {"name": "subSuite", "value": "TestLingo"}, {"name": "host", "value": "gioddembp"}, {"name": "thread", "value": "26968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allured"}]}